src/public/modules/forms/helpers/CsvMergedHeadersGenerator.ts

```diff
@@ -27,25 +38,32 @@ class CsvMergedHeadersGenerator extends CsvGenerator {
   /**
    * Returns current length of CSV file excluding header and meta-data
    */
-  length() {
+  length(): number {
     return this.unprocessed.length
   }
 
   /**
-   *
+   * Adds an UnprocessedRecord to this.unprocessed
    * @param {Object} decryptedContent
```

Comment:
we should remove the JSdocs types, since typescript code already contains more specific types. no point maintaining the type documentation in two places